### PR TITLE
Add native react Tooltip component

### PR DESCRIPTION
**Pull request by Tankenstein**
And make the `Stepper` use it. 

image

image

### CODE REVIEW

File: src/stepper/Stepper.less [hunk 1]
```diff
@@ -77,57 +35,14 @@
...
-    content: '';
-    display: block;
-    position: absolute;
 
-    margin-left: auto;
-    margin-right: auto;
-    left: 0;
-    right: 0;
-    top: -8px;
+  & .tooltip {
```
Comment (oliverviljamaa) [inline src/stepper/Stepper.less]: Don't you want to namespace this? Otherwise some external styles could apply to this, `tooltip` is pretty general.

Comment (Tankenstein) [reply src/stepper/Stepper.less]: It's a tooltip inside of a Stepper. The only place where a tooltip makes sense inside of a stepper is the label, where it's already defined by the component

Comment (oliverviljamaa) [reply src/stepper/Stepper.less]: Oh, it's the Bootstrap class name, got it then.

File: src/tooltip/Tooltip.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        top: `${(parent.offsetTop + (parent.offsetHeight / 2)) - (tooltip.offsetHeight / 2)}px`,
+        left: `${(parent.offsetLeft + parent.offsetWidth) + offset}px`,
+      };
+    case Position.BOTTOM:
+      return {
+        top: `${(parent.offsetTop + tooltip.offsetHeight) + offset}px`,
+        left: `${(parent.offsetLeft + (parent.offsetWidth / 2)) - (tooltip.offsetWidth / 2)}px`,
+      };
+    default:
+      return {};
```
Comment (coolsebz) [inline src/tooltip/Tooltip.js]: Why not add a sensible default? 🤔

**Pull request merged**